### PR TITLE
9 feature add chat message on user joining and leaving room

### LINKED ISSUE #6: Put message in chat room when users join and leave

### PR DESCRIPTION
**Pull request by nmarklund10**
fix #6

### CODE REVIEW

File: app/channels/room_channel.rb [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
 
-  def subscribed
-    # stream_from "some_channel"
-    # stream_from User.where('room_id = ?', params[:room_id])
-    stream_for room
-  end
-  
-  def connect
-    self.room_id = User.find_by(id: cookies.encrypted[:user_id])[:room_id]
-  end
```
Comment (kraig9) [inline app/channels/room_channel.rb:13]: remember this part if our broadcast stops working for some reason.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,22 @@
 class ApplicationController < ActionController::Base
     protect_from_forgery with: :exception
+
+    def send_message(username, userroom, is_leave_message=false)
+        time = Time.now.to_i
+        chat = "#{username} joined the couch!"
+        puts 'REEEEEEEEEEEEEEEEEEEEEEEEE'
```
Comment (kraig9) [inline app/controllers/application_controller.rb:7]: Take this out before commit?

File: app/channels/room_channel.rb (hunk 1 above)
Comment (nmarklund10) [reply app/channels/room_channel.rb:13]: good thing we have source control to look at previous versions of the code

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (nmarklund10) [reply app/controllers/application_controller.rb:7]: nah it gives the code charm

**Pull request merged**